OrbitClientModel/CaptureSerializer.cpp

```diff
@@ -34,6 +39,23 @@ void WriteMessage(const google::protobuf::Message* message,
   message->SerializeToCodedStream(output);
 }
 
+std::string GetCaptureFileName(const CaptureData& capture_data) {
+  time_t timestamp = std::chrono::system_clock::to_time_t(capture_data.capture_start_time());
+  std::string result;
+  result.append(Path::StripExtension(capture_data.process_name()));
```

Comment:
Not a large difference, but a little more concise with absl::StrCat:
`std::string result = absl::StrCat(Path::StripExtension(...), "_", OrbitUtils::FormatTime(timestamp));`

Reply:
Done, thanks for the suggestion!